### ISSUE TITLE
test_AngularDistribution failure

### ISSUE THREAD

**Issue by orbeckst** [MDAnalysis/mdanalysis#1756]
### Expected behaviour
analysis/test_waterdynamics:test_AngularDistribution should pass as at his before

### Actual behaviour
Now fails for some test configurations:
```
_____________________________________________ test_AngularDistribution _____________________________________________

universe = <Universe with 15 atoms>

    def test_AngularDistribution(universe):
        ad = MDAnalysis.analysis.waterdynamics.AngularDistribution(universe,
                                                                   SELECTION1,
                                                                   40)
        ad.run()
>       assert str(ad.graph[0][39]) == str("0.951172947884 0.48313682125")
E       AssertionError: assert '0.9511729478...3682125002877' == '0.951172947884 0.48313682125'
E         - 0.951172947883606 0.48313682125002877
E         ?              ^^^^              ------
E         + 0.951172947884 0.48313682125
E         ?              ^

MDAnalysisTests/analysis/test_waterdynamics.py:70: AssertionError
----------------------------------------------- Captured stderr call -----------------------------------------------
Step    10/10 [100.0%]
```

### Code to reproduce the behaviour
```
pytest -v MDAnalysisTests/analysis/test_waterdynamics.py
```
under Python 3.6

### Currently version of MDAnalysis:
(run `python -c "import MDAnalysis as mda; print(mda.__version__)"`)
latest develop

**Comment by orbeckst** [comment]
Actually, @jmborr addressed this issue in 8ef999c as part of the merged PR #1733 .

(In my PR #1757 I slightly rewrote his solution and left the test accuracy to the default for `assert_almost_equal` (i.e., `decimal=7`) instead of 12 (which we had discussed previously https://github.com/MDAnalysis/mdanalysis/pull/1733#issuecomment-357476892 but which I now consider too high in the context of this test).)

**Comment by orbeckst** [comment]
I am closing this issue but I still would like the PR to get merged.
